src/h_vialib/client.py

```diff
@@ -1,13 +1,21 @@
 """Helper classes for clients using Via proxying."""
 
 import re
+from enum import Enum
+from typing import Optional
 from urllib.parse import parse_qsl, urlencode, urlparse
 
 from webob.multidict import MultiDict
 
 from h_vialib.secure import Encryption, ViaSecureURL
 
 
+class ContentType(str, Enum):
```

Comment:
This is backwards compatible with the client code but also provides a better interface than having to guess the string values.

Reply:
So we can do `content_type == ContentType.PDF` but `content_type == "pdf"` should still work as well :+1:

Soon we'll be able to use [`StrEnum`](https://docs.python.org/3/library/enum.html#enum.StrEnum) for this.